### DISCUSSION TITLE
Accessing assets in Nuxt Layer

### DISCUSSION THREAD

**Discussion by mkalinski93** [nuxt/nuxt#26194] [Questions]
hey,

How can I access my assets inside of a nuxt layer? I saw some comments about this [here](https://github.com/nuxt/nuxt/issues/13367) but didn´t seem to work.

I´m trying to build a layer that has some .svg files that I want to use inside/outside the layer.

~layer/assets/icons/icon.svg
~assets/icons/icon.svg
./assets/icons/icon.svg
/assets/icons/icon.svg
assets/icons/icon.svg
./icons/icon.svg
/icons/icon.svg

nothing seems to work here. The only quick fix is to put the files into the public folder - but that would make the asset folder trivial inside layers. Anyone else has run into the same problematique?

**Comment by leosin** [comment]
use this, it works
```js
import avatar from '@myname/layer-base/assets/icons/base/avatar.svg'
```
